app/policies/jurisdiction_policy.rb

```diff
@@ -20,7 +20,7 @@ def create?
   end
 
   def update?
-    user.super_admin? || (user.staff? && user.jurisdictions.find(record.id))
+    user.super_admin? || (user.staff? && user.jurisdictions.find(record.id)) || user.technical_support
```

Comment:
You want to allow tech support to modify any jurisdiction, even if they are not assigned to it?

Reply:
Staff could use this to invite themself as tech support, and then give themselves access to update any jurisdiction, thus defeating the purpose of jurisdiciton membership in the first place